src/modbus.cpp

```diff
@@ -235,7 +243,7 @@ void modbus::LoadInverterConfigFromJson() {
 
   File regfile = LittleFS.open("/regs/"+this->InverterType.filename);
   if (!regfile) {
-    dbg.println("failed to open ModbusConfig.json file for writing");
+    dbg.printf("failed to open /regs/%s file\n", this->InverterType.filename.c_str());
```

Comment:
ist schon in der aktuellen dev-branch behoben

Reply:
ok, haben wir zeitgleich gefunden...